terraform/main.tf

```diff
@@ -72,6 +72,8 @@ module "aws_deploy-eu-central-1" {
   providers = {
     aws = "aws.eu-central-1"
   }
+
+  depends_on = ["${module.aws_deploy-ap-southeast-1.static_node_ips}"]
```

Comment:
From terraform/main.tf I see blue as:
aws_deploy-ap-southeast-1
aws_deploy-eu-central-1

So this makes sense.